README.md

```diff
@@ -1,36 +1,93 @@
+<img
+    src='./assets/docker.png'
+    alt='docker logo'
+    width="125px"
+/>
 
-`theiaide/theia` image contains an example of Theia based IDE for Web Developers.
+| Image Name | Description | Documentation |
+|:---|:---|:---|
+| [theiaide/theia](https://hub.docker.com/r/theiaide/theia) | Theia-based JavaScript/TypeScript (Web Technologies) example application | |
+
+_Other Variants_:
+
+| Image Name | Description | Documentation |
+|:---|:---|:---|
+| [theiaide/theia-cpp](https://hub.docker.com/r/theiaide/theia-cpp) | Theia-based C/C++ example application | [docs](./theia-cpp-docker/README.md) |
+| [theiaide/theia-dart](https://hub.docker.com/r/theiaide/theia-dart) | Theia-based Dart example application | |
+| [theiaide/theia-full](https://hub.docker.com/r/theiaide/theia-full) | Theia-based example application with all available published applications | |
+| [theiaide/theia-go](https://hub.docker.com/r/theiaide/theia-go) | Theia-based Go example application | [docs](./theia-go-docker/README.md) |
+| [theiaide/theia-java](https://hub.docker.com/r/theiaide/theia-java) | Theia-based Java example application | |
+| [theiaide/theia-python](https://hub.docker.com/r/theiaide/theia-python) | Theia-based Python example application | [docs](./theia-python-docker/README.md) |
+| [theiaide/theia-php](https://hub.docker.com/r/theiaide/theia-php) | Theia-based PHP example application | [docs](./theia-php-docker/README.md) |
+| [theiaide/theia-ruby](https://hub.docker.com/r/theiaide/theia-ruby) | Theia-based Ruby example application | |
+| [theiaide/theia-rust](https://hub.docker.com/r/theiaide/theia-rust) | Theia-based Rust example application | [docs](./theia-rust-docker/README.md) |
+| [theiaide/theia-swift](https://hub.docker.com/r/theiaide/theia-swift) | Theia-based Swift example application | |
+| [theiaide/yangster](https://hub.docker.com/r/theiaide/yangster) | Theia-based JavaScript/TypeScript (Web Technologies) example application | |
+
+---
+
+<br />
+
+### Additional Docker Examples
+
+| Image Name | Description | Documentation |
+|:---|:---|:---|
+| `theia-deb-build-docker` | Example on how to package the IDE into a Debian package | [docs](./theia-deb-build-docker/README.md) |
+| `theia-https-docker` | Example on how to add security layer over existing images | [docs](./theia-https-docker/README.md) |
+| `theia-openshift-docker` | Example image for OpenShift | |
+| `theia-rpm-build-docker` | Example on how to package the IDE into an RPM | [docs](./theia-rpm-build-docker/README.md) |
+
+<br />
 
 ### How to use `theiaide/theia` image?
 
-At the moment Theia is still in [active development](https://github.com/theia-ide/theia#roadmap). It is recommended to use [`theiaide/theia:next`](#typefoxtheianext) image to have a look at the current state.
+At the moment Theia is still in [active development](https://github.com/theia-ide/theia#roadmap). It is recommended to use `theiaide/theia:next` image to have a look at the current state.
```

Comment:
"At the moment Theia is still" -> "Theia is actively being developed"

BTW the link is the old org but anyway this section does not exist in the README anymore -> remove link

Reply:
Also "It is recommended to use `theiaide/theia:next` image to have a look at the current state." -> 
"It is recommended to use `theiaide/theia:latest` if you want the latest stable release of Theia or `theiaide/theia:next` if you want to use the most recent version of Theia at the time the image was built (bleeding edge)"